pkg/util/node/node_test.go

```diff
@@ -17,6 +17,9 @@ limitations under the License.
 package node
 
 import (
+	"net"
```

Comment:
I think this is an unnecessary import - I "believe" removing this should cause the test suite (or at least the verify test) to pass.

Reply:
Thanks @mattjmcnaughton! Updated it